nibabel/cmdline/tests/test_roi.py

```diff
@@ -143,7 +143,7 @@ def test_entrypoint(capsys):
         except SystemExit:
             pass
         else:
-            assert False, 'argparse exits on --help. If changing to another parser, update test.'
+            pytest.fail('argparse exits on --help. If changing to another parser, update test.')
```

Comment:
I think the whole `try/except/else` block should be:

```py
with pytest.raises(SystemExit):
    main()
```

Reply:
I (think I) applied your suggestion.